src/index.ts

```diff
@@ -282,3 +282,59 @@ export const gt = (versionA: string | null, versionB: string | null): boolean =>
 export const lt = (versionA: string | null, versionB: string | null): boolean => {
 	return compare(versionA, versionB) < 0;
 };
+
+/**
+ * @summary Check if a version satisfies a range
+ * @name satisfies
+ * @public
+ * @function
+ *
+ * @description Return true if the version satisfies the range
```

Comment:
So it could be something like `Returns true when the parsed version satisfies the range`?